__tests__/endpoints.test.js

```diff
@@ -198,6 +198,28 @@ describe('Patch: /api/articles/:article_id', () => {
         })
     })
 })
+describe(`delete  /api/comments/:comment_id`, () => {
+    test(`204 with no content after the comment has been deleted`, () => {
+        return request(app).delete(`/api/comments/2`).expect(204)
+        .then((res) => {
+            expect(res.body).toEqual({})
```

Comment:
That empty object is not coming from express, it's from supertest. Shouldn't test for that. Only for status code.